package.json

```diff
@@ -27,57 +31,73 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
+    "bunsen-core": "0.18.1",
     "ember-async-image": "0.1.1",
     "ember-block-slots": "^1.1.0",
-    "ember-browserify": "1.1.11",
-    "ember-cli": "~2.5.0",
+    "ember-bunsen-core": "0.12.1",
+    "ember-browserify": "1.1.13",
+    "ember-cli": "~2.8.0",
     "ember-cli-app-version": "^1.0.0",
     "ember-cli-autoprefixer": "0.6.0",
     "ember-cli-babel": "^5.1.5",
-    "ember-cli-blanket": "0.9.4",
+    "ember-cli-code-coverage": "0.3.5",
     "ember-cli-content-security-policy": "^0.5.0",
     "ember-cli-dependency-checker": "^1.2.0",
     "ember-cli-htmlbars-inline-precompile": "^0.3.1",
     "ember-cli-ic-ajax": "^0.2.4",
     "ember-cli-import-polyfill": "0.2.0",
     "ember-cli-inject-live-reload": "^1.4.0",
-    "ember-cli-mirage": "^0.1.11",
+    "ember-cli-mirage": "0.2.3",
     "ember-cli-mocha": "^0.10.1",
-    "ember-cli-moment-shim": "2.0.0",
+    "ember-cli-moment-shim": "2.2.0",
     "ember-cli-notifications": "^3.2.0",
     "ember-cli-uglify": "^1.2.0",
+    "ember-code-snippet": "1.8.0",
     "ember-computed-decorators": "0.2.2",
-    "ember-concurrency": "0.7.8",
-    "ember-data": "~2.5.1",
+    "ember-concurrency": "0.7.15",
+    "ember-data": "~2.8.1",
     "ember-disable-proxy-controllers": "^1.0.1",
+    "ember-elsewhere": "0.4.1",
     "ember-export-application-global": "^1.0.5",
-    "ember-frost-bunsen": "^6.0.0",
-    "ember-frost-core": "0.24.0",
-    "ember-frost-info-bar": "^3.0.0",
-    "ember-frost-list": "0.7.4",
-    "ember-frost-sort": "2.1.1",
-    "ember-hook": "1.3.1",
+    "ember-frost-bunsen": "12.2.6",
+    "ember-frost-core": "0.29.1",
+    "ember-frost-fields": "^3.0.0",
```

Comment:
We will have to update most of the frost dependencies to match the LTS. We should be able to provide a list of versions soon.